recipes/natural_language_processing/chatbot/app/chatbot_ui.py

```diff
@@ -67,7 +67,7 @@ def memory():
     memory = ConversationBufferWindowMemory(return_messages=True,k=3)
     return memory
 
-model_name = "" 
```

Comment:
A better approach might be to add in a selector for the model like was done for Ollama, but I wanted to favour a smaller and simpler change to start with